packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/data_sources/remote/http_clients/api_http_client.dart

```diff
@@ -36,4 +36,21 @@ class ApiHttpClient with DioMixin implements Dio {
       analyticsInterceptor,{{/analytics}}
     ]);
   }
+
+  @override
+  Future<Response> download(
```

Comment:
Do we really need this method ?

Reply:
I don't think it's necessary to include an unimplemented version of the method. Removing it...